x-pack/plugins/cases/public/methods/get_rule_id_from_event.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_RULE_NAME, ALERT_RULE_UUID } from '@kbn/rule-data-utils';
+import { get } from 'lodash/fp';
+import { Ecs } from '../../common';
+
+type Maybe<T> = T | null;
+interface TimelineItem {
```

Comment:
nit: `Timeline` is related to security solution. Maybe we can rename it to something more generic.

Reply:
yes! I want to do that, what do you think this should be? I can't figure a better name.

Reply:
I am not good at naming, hahaha. Maybe `Event` and `EventData`?

Reply:
I went for Event.